src/qililab/instruments/awg_analog_digital_converter.py

```diff
@@ -38,6 +38,7 @@ class AWGAnalogDigitalConverterSettings(AWG.AWGSettings):
 
         acquisition_delay_time: int  # ns
         awg_sequencers: Sequence[AWGADCSequencer]
+        time_of_flight: int
```

Comment:
There is also an `acquisition_delay_time` attribute in the same class. What is the difference?

Reply:
that is a delay between the last control pulse and the readout pulse. TOF is the delay between sending the pulse and startting integration